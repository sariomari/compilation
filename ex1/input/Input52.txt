] 	-))926
/*}]);?;}*/;}34
/**/voidvoid,}}25
-738
{552
-[20
int.int8
:=[ufQM8x:=+87
}string}}/*?QwF;cyS!
w*/<,void--int+	=]]:=6
,""int<)]string<G;61
-nil:=(}+	64
3492
intvoidint		"".int//2)/g,void+string2795
[;/**/=.,392
5
71
(voidvN[=:=4
6674

/*9;q;}]!!(*/int,}<NW}, -{}-)=8
+(""(]int(//?74s}(?;
);
UgWg6>. 437
	+;//
EaYa5Em7E81eC/*!
*//*!?*/7
++}/*L}]*/[88
void3960
)-"""P"string;)}+25
<<int	-string4
3550
3
//?N]9
/*?!*/)<<"Fh"voidvoid(//!N]!34}]]
)int
:=7
Dq8cW24578
4
-81
"LpEUtG"=257
510
;7
[intint{stringdkEb0pDOjF501
//;WD]j;;1!
nilint1740
;
<- <93
6
=-
6747
+/*!;1;;)/)j*/)""3
} ][;(J,262
1680
4
"pn"void[}string]//8784
(=:=int
.)}305
6897
.(-u4"U"/*7
*/void/*??]2*/}7
void9324
//;/
//
-;,[{
61
=2289
>5
"k"int/**/149
=z4
u>++	}[6576
;+
)[//!(]r;?.(7}X
371
-1894
,]/*})]z}!;*/}void)=};.void4
.void<782
"P"int]"ISqK"void;;stringstring//?];3]v;];k}]
>802
8394
{[7040
//!fint"wuPjj";[.
349
[<<-/*
]]*/v0251	.void><=5184
,"w">-/**/[int--+[206
;)+<IAC63
9972
//
752
}="WUs"nil):=""string+2
<f17
}{3063
};4
+-),/*?]!*/<4
//}}
:=287
[ 1237
[{<<(,--void]+),/**/}pQO;string-{nil:=}239
)string"""KddrbDN"8516
//
>"b"79
5
)Fint-int{(>/**/:=+979
5102
[,
int}>""M}-string.=971
88
-6564
.:="c",nil
90
}nil+151
=-=string=xtPm:=//!Q!S!C(EM}o]}))}()(!7n}!];
/*Pc]]!]
*/}{<+}intnil.)	-,"vYH""M";/*!??*/,;""}=:=void[>>]>>,>zgint:=5395
=>""[//5D)P5)F{:="N"string89
:=84
=>;+)nilint<FB85nil{)(int/*(*/(}>string-:=,"wYJM"-+string2
nil633
int/*)!
g)Z?n
!]*/c8q49631
nil
6
=6
){void>:==	:=-==:=//?}
;intint;-;void<M7:=.8821
680
,>	(=98
1
3785
3937
>(void6086
{//Q)
]:=45
int;:=.1448
=>3514
]:=+<{"":=)string"mb"}//G(intint.int,[{<=int3734
Tdwf0Qf5--}//9
);""	(.string>//4))>nil56
;,"fN"//]])(m7D
string,516
/**/))=)]intint}4322
CQLMDint{+)}(+]}208
+:=;/*;??]
*//*.}*/.
:=<+.739
-}{=.//)ostring=+intJHH:=}string{61
9
34
+.<{89
3113
[9
+=36
]+//!sK(}	;1
+"KNdrxty"(
J]4891
//
]nilnil8440
] ([)nil:=</*!Z!)9T]]}?/]}]8?}W.U!}}!;w5)u?];X;)A}5eO;
?!}!)}
!]);5}?!qq;
]?N(
a}1;!(;?

!6]*/):=6
"LqhbsI":=.[-/*;;
]
;(r*/stringint<0
<nil[(nil(//5!?}3
"e"5923
:=:=89
//;;/*T.!(?
7]h)(?(!]*/{/*
(]]oTy/!]6]*/,;intvoid8
]""nil-r:=.>string.0
<-518
,2
)./*;?(]*/}{>voidG(=312
,}97
.]/**/452
>//?7]7;?!???;?]
}-7796
]string])	string+:=//65]Zw;;?)0);

56
int=>;=>6
 [.+4
-}//
.=,nil>string+[///H/(nil73
-.f9/*!!(7m!t3
aLC!])]
?N*/{void{;]o8637
"UGi"(=9521
	10
}819
>nil <8
stringnil7545

void,""{)nilvoid/*}]?c}*/[:=nil-K1Oint(<778
; -void-1
<:=int86
,void<<string(
void[:=1803
597
{=nil>.7
:={/*])F]
*/NIint165
nil-4
;	=975
956

{,300
+>S487
),
=