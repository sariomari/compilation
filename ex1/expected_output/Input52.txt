RBRACK[1,1]
MINUS[1,4]
RPAREN[1,5]
RPAREN[1,6]
INT(926)[1,7]
SEMICOLON[3,10]
RBRACE[3,11]
INT(34)[3,12]
ID(voidvoid)[4,5]
COMMA[4,13]
RBRACE[4,14]
RBRACE[4,15]
INT(25)[4,16]
MINUS[5,1]
INT(738)[5,2]
LBRACE[6,1]
INT(552)[6,2]
MINUS[7,1]
LBRACK[7,2]
INT(20)[7,3]
TYPE_INT[9,1]
DOT[9,4]
ID(int8)[9,5]
ASSIGN[10,1]
LBRACK[10,3]
ID(ufQM8x)[10,4]
ASSIGN[10,10]
PLUS[10,12]
INT(87)[10,13]
RBRACE[11,1]
TYPE_STRING[11,2]
RBRACE[11,8]
RBRACE[11,9]
LT[12,4]
COMMA[12,5]
TYPE_VOID[12,6]
MINUS[12,10]
MINUS[12,11]
TYPE_INT[12,12]
PLUS[12,15]
EQ[12,17]
RBRACK[12,18]
RBRACK[12,19]
ASSIGN[12,20]
INT(6)[12,22]
COMMA[13,1]
STRING("")[13,2]
TYPE_INT[13,4]
LT[13,7]
RPAREN[13,8]
RBRACK[13,9]
TYPE_STRING[13,10]
LT[13,16]
ID(G)[13,17]
SEMICOLON[13,18]
INT(61)[13,19]
MINUS[14,1]
NIL[14,2]
ASSIGN[14,5]
LPAREN[14,7]
RBRACE[14,8]
PLUS[14,9]
INT(64)[14,11]
INT(3492)[15,1]
ID(intvoidint)[16,1]
STRING("")[16,13]
DOT[16,15]
TYPE_INT[16,16]
COMMA[17,1]
TYPE_VOID[17,2]
PLUS[17,6]
ID(string2795)[17,7]
LBRACK[18,1]
SEMICOLON[18,2]
EQ[18,7]
DOT[18,8]
COMMA[18,9]
INT(392)[18,10]
INT(5)[19,1]
INT(71)[20,1]
LPAREN[21,1]
ID(voidvN)[21,2]
LBRACK[21,8]
EQ[21,9]
ASSIGN[21,10]
INT(4)[21,12]
INT(6674)[22,1]
TYPE_INT[25,14]
COMMA[25,17]
RBRACE[25,18]
LT[25,19]
ID(NW)[25,20]
RBRACE[25,22]
COMMA[25,23]
MINUS[25,25]
LBRACE[25,26]
RBRACE[25,27]
MINUS[25,28]
RPAREN[25,29]
EQ[25,30]
INT(8)[25,31]
PLUS[26,1]
LPAREN[26,2]
STRING("")[26,3]
LPAREN[26,5]
RBRACK[26,6]
TYPE_INT[26,7]
LPAREN[26,10]
RPAREN[27,1]
SEMICOLON[27,2]
ID(UgWg6)[28,1]
GT[28,6]
DOT[28,7]
INT(437)[28,9]
PLUS[29,2]
SEMICOLON[29,3]
ID(EaYa5Em7E81eC)[30,1]
INT(7)[31,9]
PLUS[33,1]
PLUS[33,2]
RBRACE[33,3]
LBRACK[33,11]
INT(88)[33,12]
ID(void3960)[34,1]
RPAREN[35,1]
MINUS[35,2]
STRING("")[35,3]
STRING("P")[35,5]
TYPE_STRING[35,8]
SEMICOLON[35,14]
RPAREN[35,15]
RBRACE[35,16]
PLUS[35,17]
INT(25)[35,18]
LT[36,1]
LT[36,2]
TYPE_INT[36,3]
MINUS[36,7]
ID(string4)[36,8]
INT(3550)[37,1]
INT(3)[38,1]
INT(9)[40,1]
RPAREN[41,7]
LT[41,8]
LT[41,9]
STRING("Fh")[41,10]
ID(voidvoid)[41,14]
LPAREN[41,22]
RPAREN[42,1]
TYPE_INT[42,2]
ASSIGN[43,1]
INT(7)[43,3]
ID(Dq8cW24578)[44,1]
INT(4)[45,1]
MINUS[46,1]
INT(81)[46,2]
STRING("LpEUtG")[47,1]
EQ[47,9]
INT(257)[47,10]
INT(510)[48,1]
SEMICOLON[49,1]
INT(7)[49,2]
LBRACK[50,1]
ID(intint)[50,2]
LBRACE[50,8]
ID(stringdkEb0pDOjF501)[50,9]
ID(nilint1740)[52,1]
SEMICOLON[53,1]
LT[54,1]
MINUS[54,2]
LT[54,4]
INT(93)[54,5]
INT(6)[55,1]
EQ[56,1]
MINUS[56,2]
INT(6747)[57,1]
PLUS[58,1]
RPAREN[59,10]
STRING("")[59,11]
INT(3)[59,13]
RBRACE[60,1]
RBRACK[60,3]
LBRACK[60,4]
SEMICOLON[60,5]
LPAREN[60,6]
ID(J)[60,7]
COMMA[60,8]
INT(262)[60,9]
INT(1680)[61,1]
INT(4)[62,1]
STRING("pn")[63,1]
TYPE_VOID[63,5]
LBRACK[63,9]
RBRACE[63,10]
TYPE_STRING[63,11]
RBRACK[63,17]
INT(8784)[64,1]
LPAREN[65,1]
EQ[65,2]
ASSIGN[65,3]
TYPE_INT[65,5]
DOT[66,1]
RPAREN[66,2]
RBRACE[66,3]
INT(305)[66,4]
INT(6897)[67,1]
DOT[68,1]
LPAREN[68,2]
MINUS[68,3]
ID(u4)[68,4]
STRING("U")[68,6]
TYPE_VOID[69,3]
RBRACE[72,5]
INT(7)[72,6]
ID(void9324)[73,1]
MINUS[76,1]
SEMICOLON[76,2]
COMMA[76,3]
LBRACK[76,4]
LBRACE[76,5]
INT(61)[77,1]
EQ[78,1]
INT(2289)[78,2]
GT[79,1]
INT(5)[79,2]
STRING("k")[80,1]
TYPE_INT[80,4]
INT(149)[80,11]
EQ[81,1]
ID(z4)[81,2]
ID(u)[82,1]
GT[82,2]
PLUS[82,3]
PLUS[82,4]
RBRACE[83,1]
LBRACK[83,2]
INT(6576)[83,3]
SEMICOLON[84,1]
PLUS[84,2]
RPAREN[85,1]
LBRACK[85,2]
INT(371)[86,1]
MINUS[87,1]
INT(1894)[87,2]
COMMA[88,1]
RBRACK[88,2]
RBRACE[89,7]
TYPE_VOID[89,8]
RPAREN[89,12]
EQ[89,13]
RBRACE[89,14]
SEMICOLON[89,15]
DOT[89,16]
ID(void4)[89,17]
DOT[90,1]
TYPE_VOID[90,2]
LT[90,6]
INT(782)[90,7]
STRING("P")[91,1]
TYPE_INT[91,4]
RBRACK[91,7]
STRING("ISqK")[91,8]
TYPE_VOID[91,14]
SEMICOLON[91,18]
SEMICOLON[91,19]
ID(stringstring)[91,20]
GT[92,1]
INT(802)[92,2]
INT(8394)[93,1]
LBRACE[94,1]
LBRACK[94,2]
INT(7040)[94,3]
TYPE_INT[96,1]
STRING("wuPjj")[96,4]
SEMICOLON[96,11]
LBRACK[96,12]
DOT[96,13]
INT(349)[97,1]
LBRACK[98,1]
LT[98,2]
LT[98,3]
MINUS[98,4]
ID(v0251)[100,4]
DOT[100,10]
TYPE_VOID[100,11]
GT[100,15]
LT[100,16]
EQ[100,17]
INT(5184)[100,18]
COMMA[101,1]
STRING("w")[101,2]
GT[101,5]
MINUS[101,6]
LBRACK[101,11]
TYPE_INT[101,12]
MINUS[101,15]
MINUS[101,16]
PLUS[101,17]
LBRACK[101,18]
INT(206)[101,19]
SEMICOLON[102,1]
RPAREN[102,2]
PLUS[102,3]
LT[102,4]
ID(IAC63)[102,5]
INT(9972)[103,1]
INT(752)[105,1]
RBRACE[106,1]
EQ[106,2]
STRING("WUs")[106,3]
NIL[106,8]
RPAREN[106,11]
ASSIGN[106,12]
STRING("")[106,14]
TYPE_STRING[106,16]
PLUS[106,22]
INT(2)[106,23]
LT[107,1]
ID(f17)[107,2]
RBRACE[108,1]
LBRACE[108,2]
INT(3063)[108,3]
RBRACE[109,1]
SEMICOLON[109,2]
INT(4)[109,3]
PLUS[110,1]
MINUS[110,2]
RPAREN[110,3]
COMMA[110,4]
LT[110,12]
INT(4)[110,13]
ASSIGN[112,1]
INT(287)[112,3]
LBRACK[113,1]
INT(1237)[113,3]
LBRACK[114,1]
LBRACE[114,2]
LT[114,3]
LT[114,4]
LPAREN[114,5]
COMMA[114,6]
MINUS[114,7]
MINUS[114,8]
TYPE_VOID[114,9]
RBRACK[114,13]
PLUS[114,14]
RPAREN[114,15]
COMMA[114,16]
RBRACE[115,5]
ID(pQO)[115,6]
SEMICOLON[115,9]
TYPE_STRING[115,10]
MINUS[115,16]
LBRACE[115,17]
NIL[115,18]
ASSIGN[115,21]
RBRACE[115,23]
INT(239)[115,24]
RPAREN[116,1]
TYPE_STRING[116,2]
STRING("")[116,8]
STRING("KddrbDN")[117,1]
INT(8516)[117,10]
GT[120,1]
STRING("b")[120,2]
INT(79)[120,5]
INT(5)[121,1]
RPAREN[122,1]
ID(F)[122,2]
TYPE_INT[123,1]
MINUS[123,4]
TYPE_INT[123,5]
LBRACE[123,8]
LPAREN[123,9]
GT[123,10]
ASSIGN[123,15]
PLUS[123,17]
INT(979)[123,18]
INT(5102)[124,1]
LBRACK[125,1]
COMMA[125,2]
TYPE_INT[126,1]
RBRACE[126,4]
GT[126,5]
STRING("")[126,6]
ID(M)[126,8]
RBRACE[126,9]
MINUS[126,10]
TYPE_STRING[126,11]
DOT[126,17]
EQ[126,18]
INT(971)[126,19]
INT(88)[127,1]
MINUS[128,1]
INT(6564)[128,2]
DOT[129,1]
ASSIGN[129,2]
STRING("c")[129,4]
COMMA[129,7]
NIL[129,8]
INT(90)[130,1]
RBRACE[131,1]
NIL[131,2]
PLUS[131,5]
INT(151)[131,6]
EQ[132,1]
MINUS[132,2]
EQ[132,3]
TYPE_STRING[132,4]
EQ[132,10]
ID(xtPm)[132,11]
ASSIGN[132,15]
RBRACE[135,3]
LBRACE[135,4]
LT[135,5]
PLUS[135,6]
RBRACE[135,7]
ID(intnil)[135,8]
DOT[135,14]
RPAREN[135,15]
MINUS[135,17]
COMMA[135,18]
STRING("vYH")[135,19]
STRING("M")[135,24]
SEMICOLON[135,27]
COMMA[135,35]
SEMICOLON[135,36]
STRING("")[135,37]
RBRACE[135,39]
EQ[135,40]
ASSIGN[135,41]
TYPE_VOID[135,43]
LBRACK[135,47]
GT[135,48]
GT[135,49]
RBRACK[135,50]
GT[135,51]
GT[135,52]
COMMA[135,53]
GT[135,54]
ID(zgint)[135,55]
ASSIGN[135,60]
INT(5395)[135,62]
EQ[136,1]
GT[136,2]
STRING("")[136,3]
LBRACK[136,5]
ID(F)[137,1]
LBRACE[137,2]
ASSIGN[137,3]
STRING("N")[137,5]
ID(string89)[137,8]
ASSIGN[138,1]
INT(84)[138,3]
EQ[139,1]
GT[139,2]
SEMICOLON[139,3]
PLUS[139,4]
RPAREN[139,5]
ID(nilint)[139,6]
LT[139,12]
ID(FB85nil)[139,13]
LBRACE[139,20]
RPAREN[139,21]
LPAREN[139,22]
TYPE_INT[139,23]
LPAREN[139,31]
RBRACE[139,32]
GT[139,33]
TYPE_STRING[139,34]
MINUS[139,40]
ASSIGN[139,41]
COMMA[139,43]
STRING("wYJM")[139,44]
MINUS[139,50]
PLUS[139,51]
ID(string2)[139,52]
ID(nil633)[141,1]
TYPE_INT[142,1]
ID(c8q49631)[145,5]
NIL[146,1]
INT(6)[147,1]
EQ[148,1]
INT(6)[148,2]
RPAREN[149,1]
LBRACE[149,2]
TYPE_VOID[149,3]
GT[149,7]
ASSIGN[149,8]
EQ[149,10]
ASSIGN[149,12]
MINUS[149,14]
EQ[149,15]
EQ[149,16]
ASSIGN[149,17]
SEMICOLON[150,1]
ID(intint)[150,2]
SEMICOLON[150,8]
MINUS[150,9]
SEMICOLON[150,10]
TYPE_VOID[150,11]
LT[150,15]
ID(M7)[150,16]
ASSIGN[150,18]
DOT[150,20]
INT(8821)[150,21]
INT(680)[151,1]
COMMA[152,1]
GT[152,2]
LPAREN[152,4]
EQ[152,5]
INT(98)[152,6]
INT(1)[153,1]
INT(3785)[154,1]
INT(3937)[155,1]
GT[156,1]
LPAREN[156,2]
ID(void6086)[156,3]
LBRACE[157,1]
RPAREN[158,1]
RBRACK[159,1]
ASSIGN[159,2]
INT(45)[159,4]
TYPE_INT[160,1]
SEMICOLON[160,4]
ASSIGN[160,5]
DOT[160,7]
INT(1448)[160,8]
EQ[161,1]
GT[161,2]
INT(3514)[161,3]
RBRACK[162,1]
ASSIGN[162,2]
PLUS[162,4]
LT[162,5]
LBRACE[162,6]
STRING("")[162,7]
ASSIGN[162,9]
RPAREN[162,11]
TYPE_STRING[162,12]
STRING("mb")[162,18]
RBRACE[162,22]
LPAREN[163,1]
ID(intint)[163,2]
DOT[163,8]
TYPE_INT[163,9]
COMMA[163,12]
LBRACK[163,13]
LBRACE[163,14]
LT[163,15]
EQ[163,16]
ID(int3734)[163,17]
ID(Tdwf0Qf5)[164,1]
MINUS[164,9]
MINUS[164,10]
RBRACE[164,11]
RPAREN[165,1]
SEMICOLON[165,2]
STRING("")[165,3]
LPAREN[165,6]
DOT[165,7]
TYPE_STRING[165,8]
GT[165,14]
RPAREN[166,1]
GT[166,2]
ID(nil56)[166,3]
SEMICOLON[167,1]
COMMA[167,2]
STRING("fN")[168,1]
TYPE_STRING[169,1]
COMMA[169,7]
INT(516)[169,8]
RPAREN[170,5]
RPAREN[170,6]
EQ[170,7]
RPAREN[170,8]
RBRACK[170,9]
ID(intint)[170,10]
RBRACE[170,16]
INT(4322)[170,17]
ID(CQLMDint)[171,1]
LBRACE[171,9]
PLUS[171,10]
RPAREN[171,11]
RBRACE[171,12]
LPAREN[171,13]
PLUS[171,14]
RBRACK[171,15]
RBRACE[171,16]
INT(208)[171,17]
PLUS[172,1]
ASSIGN[172,2]
SEMICOLON[172,4]
DOT[173,9]
ASSIGN[174,1]
LT[174,3]
PLUS[174,4]
DOT[174,5]
INT(739)[174,6]
MINUS[175,1]
RBRACE[175,2]
LBRACE[175,3]
EQ[175,4]
DOT[175,5]
TYPE_STRING[176,1]
EQ[176,7]
PLUS[176,8]
ID(intJHH)[176,9]
ASSIGN[176,15]
RBRACE[176,17]
TYPE_STRING[176,18]
LBRACE[176,24]
INT(61)[176,25]
INT(9)[177,1]
INT(34)[178,1]
PLUS[179,1]
DOT[179,2]
LT[179,3]
LBRACE[179,4]
INT(89)[179,5]
INT(3113)[180,1]
LBRACK[181,1]
INT(9)[181,2]
PLUS[182,1]
EQ[182,2]
INT(36)[182,3]
RBRACK[183,1]
PLUS[183,2]
SEMICOLON[184,2]
INT(1)[184,3]
PLUS[185,1]
STRING("KNdrxty")[185,2]
LPAREN[185,11]
ID(J)[186,1]
RBRACK[186,2]
INT(4891)[186,3]
RBRACK[188,1]
ID(nilnil8440)[188,2]
RBRACK[189,1]
LPAREN[189,3]
LBRACK[189,4]
RPAREN[189,5]
NIL[189,6]
ASSIGN[189,9]
LT[189,11]
RPAREN[199,6]
ASSIGN[199,7]
INT(6)[199,9]
STRING("LqhbsI")[200,1]
ASSIGN[200,9]
DOT[200,11]
LBRACK[200,12]
MINUS[200,13]
ID(stringint)[203,6]
LT[203,15]
INT(0)[203,16]
LT[204,1]
NIL[204,2]
LBRACK[204,5]
LPAREN[204,6]
NIL[204,7]
LPAREN[204,10]
INT(3)[205,1]
STRING("e")[206,1]
INT(5923)[206,4]
ASSIGN[207,1]
ASSIGN[207,3]
INT(89)[207,5]
LBRACE[210,12]
COMMA[211,14]
SEMICOLON[211,15]
ID(intvoid8)[211,16]
RBRACK[212,1]
STRING("")[212,2]
NIL[212,4]
MINUS[212,7]
ID(r)[212,8]
ASSIGN[212,9]
DOT[212,11]
GT[212,12]
TYPE_STRING[212,13]
DOT[212,19]
INT(0)[212,20]
LT[213,1]
MINUS[213,2]
INT(518)[213,3]
COMMA[214,1]
INT(2)[214,2]
RPAREN[215,1]
DOT[215,2]
RBRACE[216,4]
LBRACE[216,5]
GT[216,6]
ID(voidG)[216,7]
LPAREN[216,12]
EQ[216,13]
INT(312)[216,14]
COMMA[217,1]
RBRACE[217,2]
INT(97)[217,3]
DOT[218,1]
RBRACK[218,2]
INT(452)[218,7]
GT[219,1]
RBRACE[220,1]
MINUS[220,2]
INT(7796)[220,3]
RBRACK[221,1]
TYPE_STRING[221,2]
RBRACK[221,8]
RPAREN[221,9]
TYPE_STRING[221,11]
PLUS[222,1]
ASSIGN[222,2]
INT(56)[224,1]
TYPE_INT[225,1]
EQ[225,4]
GT[225,5]
SEMICOLON[225,6]
EQ[225,7]
GT[225,8]
INT(6)[225,9]
LBRACK[226,2]
DOT[226,3]
PLUS[226,4]
INT(4)[226,5]
MINUS[227,1]
RBRACE[227,2]
DOT[228,1]
EQ[228,2]
COMMA[228,3]
NIL[228,4]
GT[228,7]
TYPE_STRING[228,8]
PLUS[228,14]
LBRACK[228,15]
LPAREN[229,1]
ID(nil73)[229,2]
MINUS[230,1]
DOT[230,2]
ID(f9)[230,3]
LBRACE[232,5]
TYPE_VOID[232,6]
LBRACE[232,10]
SEMICOLON[232,11]
RBRACK[232,12]
ID(o8637)[232,13]
STRING("UGi")[233,1]
LPAREN[233,6]
EQ[233,7]
INT(9521)[233,8]
INT(10)[234,2]
RBRACE[235,1]
INT(819)[235,2]
GT[236,1]
NIL[236,2]
LT[236,6]
INT(8)[236,7]
ID(stringnil7545)[237,1]
TYPE_VOID[239,1]
COMMA[239,5]
STRING("")[239,6]
LBRACE[239,8]
RPAREN[240,1]
ID(nilvoid)[240,2]
LBRACK[240,18]
ASSIGN[240,19]
NIL[240,21]
MINUS[240,24]
ID(K1Oint)[240,25]
LPAREN[240,31]
LT[240,32]
INT(778)[240,33]
SEMICOLON[241,1]
MINUS[241,3]
TYPE_VOID[241,4]
MINUS[241,8]
INT(1)[241,9]
LT[242,1]
ASSIGN[242,2]
ID(int86)[242,4]
COMMA[243,1]
TYPE_VOID[243,2]
LT[243,6]
LT[243,7]
TYPE_STRING[243,8]
LPAREN[243,14]
TYPE_VOID[244,1]
LBRACK[244,5]
ASSIGN[244,6]
INT(1803)[244,8]
INT(597)[245,1]
LBRACE[246,1]
EQ[247,1]
NIL[247,2]
GT[247,5]
DOT[247,6]
INT(7)[247,7]
ASSIGN[248,1]
LBRACE[248,3]
ID(NIint165)[250,3]
NIL[251,1]
MINUS[251,4]
INT(4)[251,5]
SEMICOLON[252,1]
EQ[252,3]
INT(975)[252,4]
INT(956)[253,1]
LBRACE[255,1]
COMMA[255,2]
INT(300)[255,3]
PLUS[256,1]
GT[256,2]
ID(S487)[256,3]
RPAREN[257,1]
COMMA[257,2]
EQ[258,1]